src/core/data.jl

```diff
@@ -1986,30 +1986,23 @@ end
 computes the connected components of the network graph
 returns a set of sets of bus ids, each set is a connected component
 """
-function connected_components(data::Dict{String,<:Any})
+function connected_components(data::Dict{String,<:Any}; lines=["branch", "dcline"])
```

Comment:
Lets call this parameter `edges` as that is the CS term for the connectors in a generic graph.